lib/taurus/qt/qtgui/graphic/taurusgraphic.py

```diff
@@ -26,9 +26,17 @@
 taurusgraphic.py:
 """
 from __future__ import print_function
+from __future__ import division
 
 # TODO: Tango-centric
 
+from future import standard_library
+standard_library.install_aliases()
+from builtins import str
+from builtins import range
+from past.builtins import basestring
```

Comment:
we should later on check all the `from past.builtins` imports and see if we can write more natural code